README.md

```
@@ -0,0 +1,112 @@
+   bun run dev
+   ```
+
+6. **Test API**:
+   ```bash
+   curl http://localhost:3000/health
+   ```
+
+## Environment Variables
+
+- `OBSIDIAN_VAULT_PATH`: Path to your Obsidian vault directory
+- `CHROMADB_URL`: ChromaDB server URL (default: http://localhost:8000)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)